### PR TITLE
Add Expr support for To/FromBase64

### LINKED ISSUE #59: Support to/fromBase64
Base64 is a commonly use and widely supported. The api should probably be
```
def fromBase64(e: Expr[String]): Expr[Option[Binary]]
def toBase64(e: Expr[Binary]): Expr[String]
```
since a string might not be valid base64.

### PR DESCRIPTION
**Pull request by eejbyfeldt**
Fixes #59

Started out as an AI task to test https://github.com/unum-io/tyda/pull/62
but ended up taking over as the AI took a really long time figuring out
the edge cases around spark producing chunked output.

### CODE REVIEW

File: tyda-sql/src/main/scala/com/choreograph/tyda/sql/SqlDialect.scala [hunk 1]
```diff
@@ -76,6 +78,33 @@ final case class SqlDialect(
...
+      */
+    case Function(name: String)
+  }
+
+  /** How to encode binary to a Base64 string. */
+  enum ToBase64Support {
+
+    /** A simple function taking the binary and encodes it.
+      *
+      * If isChucked is true it produces a values chunked by newlines with needs
```
Comment (dahlbaek) [inline tyda-sql/src/main/scala/com/choreograph/tyda/sql/SqlDialect.scala]:
```suggestion
      * If isChunked is true it produces a values chunked by newlines with needs
```

File: tyda-sql/src/main/scala/com/choreograph/tyda/sql/SqlDialect.scala [hunk 2]
```diff
@@ -76,6 +78,33 @@ final case class SqlDialect(
...
+      * to be filtered out to have correct behaviror.
```
Comment (dahlbaek) [inline tyda-sql/src/main/scala/com/choreograph/tyda/sql/SqlDialect.scala]:
```suggestion
      * to be filtered out to have correct behavior.
```

**Review by dahlbaek — DISMISSED** [review]

**Review by dahlbaek — APPROVED** [review]
